templates/ts/sso-tab-with-obo-flow/README.md

```diff
@@ -0,0 +1,68 @@
+# Getting Started with Hello World Tab with Backend Sample (Azure)
```

Comment:
I think this readme is for sample. You may want to replace it with the readme for template.
https://github.com/OfficeDev/TeamsFx/pull/9052

Reply:
Sure. Junjie said he will update the readme for tabs, and I will leverage with his changes after his pr is merged.